testing/amule/APKBUILD

```diff
@@ -2,33 +2,25 @@
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=amule
 pkgsvn=aMule-SVN-r
```

Comment:
This is not a standard abuild variable, so it must be prefixed with an underscore (`_pkgsvn`).
I would prefer to name it `_pkgname` instead, that's kinda a convention in abuilds.